.github/workflows/build.yml

```
@@ -456,6 +456,112 @@ jobs:
           ../scripts/xcodebuild.sh ${{ steps.configure.outputs.project-directory }}/TemplateExample.xcworkspace build
         working-directory: template-example
     timeout-minutes: 60
+  visionos:
+    name: "visionOS"
+    runs-on: macos-14
```

Do we need to build visionOS nightly?
```suggestion
    runs-on: macos-14
    if: ${{ github.event_name != 'schedule' }}
```

---

It doesn't seem like callstack publishes nightlies.

---

@thymikee should we publish nightlies? I thought it was a bit too early since upstream is still breaking visionOS regularly.

---

For now, let's not build visionOS nightly.

---

In a future change, perhaps could build RNM nightlies. For now, I'll commit the suggestion.